soccer/motion/MotionControl.cpp

```diff
@@ -33,7 +33,7 @@ void MotionControl::createConfiguration(Configuration* cfg) {
 MotionControl::MotionControl(OurRobot* robot) : _angleController(0, 0, 0, 50) {
     _robot = robot;
 
-    _robot->radioTx.set_robot_id(_robot->shell());
+    _robot->robotPacket->set_uid(_robot->shell());
```

Comment:
We'll have to figure out how we want to handle the difference between uid and shell id.